packages/n3h-mock/lib/index.js

```diff
@@ -212,47 +216,47 @@ class N3hMock extends AsyncClass {
 
   _getMemRef (dnaAddress) {
     if (!(dnaAddress in this._memory)) {
-      const mem = new Mem()
-      mem.registerIndexer((store, hash, data) => {
-        if (data && data.type === 'dht') {
-          this._ipc.send('json', {
-            method: 'storeDht',
-            _id: data._id,
-            dnaAddress,
-            agentId: data.agentId,
-            address: data.address,
-            content: data.content
-          })
-        }
-      })
-      mem.registerIndexer((store, hash, data) => {
-        if (data && data.type === 'dhtMeta') {
-          log.e('got dhtMeta', data)
+    const mem = new Mem()
+    mem.registerIndexer((store, hash, data) => {
+      if (data && data.type === 'dht') {
+        this._ipc.send('json', {
+          method: 'handleStoreDht',
+          _id: data._id,
+          dnaAddress,
+          agentId: data.agentId,
+          address: data.address,
+          content: data.content
+        })
+      }
+    })
+    mem.registerIndexer((store, hash, data) => {
+      if (data && data.type === 'dhtMeta') {
+        log.e('got dhtMeta', data)
+        this._ipc.send('json', {
+          method: 'handleStoreDhtMeta',
+          _id: data._id,
+          dnaAddress,
+          agentId: data.agentId,
+          address: data.address,
+          fromAgentId: data.fromAgentId,
+          attribute: data.attribute,
+          content: data.content
+        })
+      }
+    })
+    this._memory[dnaAddress] = {
+      mem,
+      agentToTransportId: mem.registerIndexer((store, hash, data) => {
+        if (data && data.type === 'agent') {
+          store[data.agentId] = data.transportId
           this._ipc.send('json', {
-            method: 'storeDhtMeta',
-            _id: data._id,
-            dnaAddress,
-            agentId: data.agentId,
-            address: data.address,
-            fromAgentId: data.fromAgentId,
-            attribute: data.attribute,
-            content: data.content
+            method: 'peerConnected',
+            dnaAddress: dnaAddress,
+            agentId: data.agentId
           })
         }
       })
-      this._memory[dnaAddress] = {
-        mem,
-        agentToTransportId: mem.registerIndexer((store, hash, data) => {
-          if (data && data.type === 'agent') {
-            store[data.agentId] = data.transportId
-            this._ipc.send('json', {
-              method: 'peerConnected',
-              dnaAddress: dnaAddress,
-              agentId: data.agentId
-            })
-          }
-        })
-      }
+    }
```

Comment:
I only changed what was in 'method' field but maybe I moved some whitespaces around :/